### PR TITLE
flow: add raw packet to flow

### PR DESCRIPTION
**Pull request by safchain**

### CODE REVIEW

File: flow/flow.proto [hunk 1]
```diff
@@ -28,50 +28,56 @@ syntax = "proto3";
...
 enum FlowProtocol {
-	ETHERNET = 0;
-	IPV4 = 1;
-	TCPPORT = 2;
-	UDPPORT = 3;
-	SCTPPORT = 4;
-	IPV6 = 5;
-	ICMPV4 = 6;
-	ICMPV6 = 7;
+  ETHERNET = 0;
```
Comment (nplanel) [inline flow/flow.proto]: why changing ident ?

File: flow/flow.proto [hunk 2]
```diff
@@ -111,16 +117,21 @@ message Flow {
...
-	string ANodeTID = 34;
-	string BNodeTID = 35;
+  string NodeTID = 33;
+  string ANodeTID = 34;
+  string BNodeTID = 35;
+
+/* raw packets, will not be exported, see Makefile */
+  repeated RawPacket LastRawPackets = 36;
+/* number of raw packet captured */
+  int64 RawPackets = 37;
```
Comment (nplanel) [inline flow/flow.proto]: s|RawPackets|RawPacketsCaptured|g

File: flow/table.go [hunk 3]
```diff
@@ -87,8 +94,9 @@ type Table struct {
 }
 
 // NewTable creates a new flow table
-func NewTable(updateHandler *Handler, expireHandler *Handler, pipeline *EnhancerPipeline) *Table {
+func NewTable(updateHandler *Handler, expireHandler *Handler, pipeline *EnhancerPipeline, nodeTID string, opts TableOpts) *Table {
```
Comment (nplanel) [inline flow/table.go]: nodeTID could be an option of opts

File: flow/flow.proto (hunk 1 above)
Comment (safchain) [inline flow/flow.proto]: it is more fixing, because there was a mix of tab/space

File: flow/flow.proto (hunk 2 above)
Comment (safchain) [inline flow/flow.proto]: Done

File: flow/table.go (hunk 3 above)
Comment (safchain) [inline flow/table.go]: OK for options as optional. For nodeTID it is not really an option as it doesnt' really change the way to flow are handled by the flow table

Comment (nplanel) [inline flow/table.go]: ok
the idea was more like : default tid would be "" if not set

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-elasticsearch

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun functional-tests-backend-orientdb

**Comment by safchain** [comment]
rerun tests

**Pull request merged**